cmd/flow-aggregator/options.go

```diff
@@ -82,24 +88,29 @@ func (o *Options) complete(args []string) error {
 			return err
 		}
 		o.config = c
+		o.setDefaults()
 	}
 	return nil
 }
 
 // validate validates all the required options.
 func (o *Options) validate(args []string) error {
+	var err error
 	if len(args) != 0 {
 		return errors.New("no positional arguments are supported")
 	}
-	if o.config.ExternalFlowCollectorAddr == "" {
-		return fmt.Errorf("IPFIX flow collector address should be provided")
+	if !o.isFlowCollectorSet() && !o.isClickHouseSet() {
+		return fmt.Errorf("external flow collector or ClickHouse should be configured")
 	}
-	host, port, proto, err := flowexport.ParseFlowCollectorAddr(o.config.ExternalFlowCollectorAddr, defaultExternalFlowCollectorPort, defaultExternalFlowCollectorTransport)
-	if err != nil {
-		return err
+	if o.config.FlowCollectorSink.Address != "" {
```

Comment:
this should probably use `isFlowCollectorSet` instead, no?
I feel like we are mostly ignoring the `enable` parameter here